eclipse-theia/theia
# fix(tabs): replace tab character by space characters

#### What it does
just replace tab characters by space

#### How to test
you may reproduce this fix just by opening and saving the file.
About test, hard to say how to test as it's like a no-op but plug-ins should still work as usual
And in github diff view, you can see it's only related to spaces/indent

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
